frontend/src/Settings/MediaManagement/Naming/NamingModal.js

```diff
@@ -47,29 +47,29 @@ const fileNameTokens = [
 ];
 
 const authorTokens = [
-  { token: '{Author Name}', example: 'Author Name' },
+  { token: '{Author Name}', example: 'Author's Name' },
```

Comment:
escape the `'` like `\'`

ref https://github.com/Radarr/Radarr/commit/b06ec1f291f90dbfcaf155847af991d7bf7af4cb